Add meetup (#740)
mainmatter/eurorust.eu: content/activities/meetup.md

+++

> title = "Rust Meetup"

Reviewer: ```suggestion
title = "Rust Paris Meetup"
```

template = "meetup.html"
[extra]
  date = "Wednesday, October 8, 2025"
  time = "19:00–22:00"
  location = "22 Rue de la Michodière, 75009 Paris"
  location_name = "amo HQ"
  location_url = "https://maps.app.goo.gl/ZES1JJUGWiPkEgey9"

>   abstract = "Rust Paris has partnered up with EuroRust and amo and will host a special get-together for all Rustaceans the evening before the conference. Join in on the to connect with other Rust enthusiasts over ample food and drink. The community of Rust Paris is looking forward to seeing you there!"

Reviewer: ```suggestion
  abstract = "Rust Paris has partnered up with EuroRust and amo and will host a get-together for all Rustaceans the evening before the conference. Join to connect with other Rust enthusiasts over talks, food and drinks. Rust Paris is looking forward to seeing you there!"
```
